### DISCUSSION TITLE
Fees for jurors and incentives

### DISCUSSION THREAD

**Discussion by luisivan** [nation3/court#22] [Specs]
While reviewing the spec one more time, two key points came to mind:
1. It's paramount that the Court can accrue value proportional to the stake it secures. It aligns incentives the right way, as it accrues more value to $NATION, which in case of an appeal becomes the last resort providing security to the parties of an agreement.
2. The Court needs to be paid arbitration fees ideally even when a dispute hasn't been raised. This is because otherwise it doesn't satisfy the first point (it would only accrue value out of disputed stake, not total stake). But most importantly, the Court only receiving fees when there are disputes means that there is a built-in incentive for the Court (and broadly Nation3) to have as many disputes as possible, which is exactly the contrary to what we should encourage. Ideally, a system of law and its enforcement is so good that disputes rarely arise. That way, Nation3 is incentivized to work on a great constitution, on using bleeding edge tooling to formalize agreements (e.g. [Catala](https://catala-lang.org)) and so forth. Jurors also need to be very qualified and ready to jump in, even if there are few disputes.

When it comes to #2, taking part of the stake as fees might not even be the best idea. Another idea could actually be for the parties to increase their stake (once in a while) in order to keep the agreement secured by the Court.

Whichever way we end up choosing, the best part is that most of the monetization model doesn't need to be encoded. In the beginning the dispute fees can just be reflected by jurors as a part of the settlement outcome (therefore can be a fixed amount or a percentage of the stake).

The only question I'd have here is whether it would make sense to also add a way for a % of the agreement stake to be sent to the DAO upon finalization of an agreement. Could be implemented in `withdrawFromAgreement `. Thoughts?

**Comment by xPi2** [comment]
I don't agree with your take on the fees.
> there is a built-in incentive for the Court (and broadly Nation3) to have as many disputes as possible, which is exactly the contrary to what we should encourage.

I think that precisely the fact that only fees are paid in the event of a dispute incentivise the users to come to friendly settlements without the need of raising a dispute.

**Reply by luisivan** [reply]
There are two sides:
- Users are disincentivized by default for a dispute to arise, because it's added overhead for them.
- Jurors/Nation3 are currently incentivized for disputes to arise, because of the points raised above.

However, a fixed fee for raising a dispute is always a good idea as a way to disincentivize users even more from doing so. But that's not incompatible with the rest of the points I raised. Curious to hear your thoughts! :D

**Reply by xPi2** [reply]
Summarizing what we have spoked in-voice and more detailed proposal:
- % fee over the total deposited funds in the agreement. They will be implemented and captured directly by the framework at withdrawal.
- Fixed cost for arbitrating the agreement. The arbitrator will include it on the settlement.

This setup will keep the incentives to avoid disputes while also provides a constant capturing of value from usage.

I will make the needed adjustments to the Collateral Agreements framework.

**Comment by aahna-ashina** [comment]
@luisivan @xPi2 Which token will be used when collecting fees?  `$NATION` or `$ETH`?

**Reply by xPi2** [reply]
I would say NATION for the current implementation, this could be extended to shares in other tokens depending on future capabilities (escrows, payment streams, etc). In any case I would say WETH instead of ETH

**Comment by xPi2** [comment]
An update to this discussion:

In line with the "tax-free" ethos of Nation3 we decided to remove the need for a "usage fee" for the collateral agreements. As this type of agreements require you to get and lock $NATION in the agreement, we expect this will generate enough demand for the token to allow us to finance operations without applying an explicit fee.

We will keep the fixed costs for arbitration when disputing an agreement and appealing a resolution.
Ideally all the fees should be collected by the arbitrator.
As the dispute is started from the agreement framework I'm still not sure where to define and apply the fees.
I'm thinking that different agreements frameworks should have different dispute costs while an appeal should be a multiplier of the dispute cost.

**Reply by xPi2** [reply]
I almost forgot, for liability / security reasons I removed the capabilitie of the arbitrator to send funds to other accounts but the joined parties so we are not going to include the fees in the settlement. So we must collect them in the moment of disputing / appealing.

**Comment by AaronAbuUsama** [comment]
coming to this a bit late but ill share my thoughts nonetheless. As I see it, there are two goals we are trying to engineer around 

1. We do not want a system that incentivises disputes 
2. We do not want to "tax" users

fees != taxes, not necessarily anyway. To me, a tax is something you pay irrespective of behaviour 
> The Court needs to be paid arbitration fees ideally even when a dispute hasn't been raised.

this to me would have been a tax, because you pay even if your behaviour is in line with the expectation. Where an Arbitration fee based on actual disputes is a fee. Also, that depends on who is paying the fee, if it's the looser of a dispute, then it's a fee, if it's both, then the party who acted inline with the agreement is being taxed. 

> the Court only receiving fees when there are disputes means that there is a built-in incentive for the Court (and broadly Nation3) to have as many disputes as possible

I dont buy this either. Yes, the court makes more money from fees, there is also the non-monetary incentive where more disputes mean a more engaged and better-trained juror pool. But I dont see a mechanism where the court can materially affect the level of disputes raised in agreements between private parties. If there are some, they are indirect, whereas the fee itself is a direct disincentive for the 'guilty' party to let a dispute be raised in the first place

**Reply by xPi2** [reply]
I agree with you that applying a fee for the good use of the jurisdiction it can be seen as a tax. That's why I would like to restrict the fees to the use of the dispute arbitration service.

Said that, I still think we should work on a way to capture value from the use of the jurisdiction, so we can keep incentivizing work towards its growth: new frameworks, agreement templates, etc. I think that generating demand towards a N3 native token could be the best tradeoff to capture value without taxing users or limiting the scope of the service.

**Reply by luisivan** [reply]
Not a fan of taxes, but fan of locking up $NATION in proportional amount to the assets secured by the agreements within the Nation3 jurisdiction. Of course the ideal would be a stablecoin backed by $NATION, but in the meantime we can find compromises.

> But I dont see a mechanism where the court can materially affect the level of disputes raised in agreements between private parties.

The Court doesn't, the jurisdiction does. The better architected the law is and the better the tooling for drafting agreements is, the fewer disputes should exist.

**Reply by AaronAbuUsama** [reply]
> The Court doesn't, the jurisdiction does. The better architected the law is and the better the tooling for drafting agreements is, the fewer disputes should exist.

Is the idea here that the Jurisdiction is incentivised to _not_ improve the tooling and law because they would get more fees because they would be more disputes? that seems counterintuitive. For that to be the case the Jurisdiction would first have had to build the system, tooling, and captured a large enough user base for that to be a realistic proposition. And then once they got there, would there not naturally be forks from other jurisdictions trying to out compete?

**Comment by AaronAbuUsama** [comment]
> Said that, I still think we should work on a way to capture value from the use of the jurisdiction...

100% with everything you just said

> but fan of locking up $NATION in proportional amount to the assets secured by the agreements  

I really like this primitive of long term locks. It is a different beast all together from staking.
